### ISSUE TITLE
Improve Highstate

### ISSUE THREAD

**Issue by cmclaughlin** [martinhoefling/molten#43]
It would be nice to be able to run highstate with test=True via Molten.  Also display the output/return.

**Comment by cmclaughlin** [comment]
Sorry, I see issue #3 now.  I'll just edit this to suggest test=True

**Comment by martinhoefling** [comment]
@cmlaughlin you think about the minion tab? You already can execute state.highstate with test=true in the kwargs.

**Comment by cmclaughlin** [comment]
Oh, maybe my installation isn't working properly.  This is what I see:

<img width="846" alt="screen shot 2017-05-19 at 9 58 17 am" src="https://cloud.githubusercontent.com/assets/1061109/26236605/e237aa2e-3c79-11e7-824f-9bc007bcc3cb.png">

In that screenshot I pressed the highstate button and the color went from blue to grey. But I don't see option for kwargs.

**Comment by martinhoefling** [comment]
Yes, in the minion tab, that's not possible (yet). But if you're using the execute tab, you can set the kwargs. In the minion tab, I could imagine the following: Highstate and Test Highstate Button (checkbox alternatively?). The result could be shown in a popover?

**Comment by cmclaughlin** [comment]
Yes, that sounds good!
